### PR TITLE
Incorperating code review comments

### PR DESCRIPTION
**Pull request by kasundsilva**

### CODE REVIEW

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java [hunk 1]
```diff
@@ -286,10 +287,7 @@
     public static final String DELETE_ALL_APP_VERSION_EVENTS =
             "Delete from AC_EVENT where version_id = (SELECT id FROM AC_VERSION WHERE hash_id=? AND tenant_id=?)";
 
-	public static final String GET_TENANT_APPLICATION_COUNT = "SELECT COUNT(*) FROM AC_APPLICATION WHERE tenant_id = ? " +
-            "AND cloud_id=(select id from AC_CLOUD WHERE name=?)";
-
-    public static final String GET_TENANT_RUNNING_CONTAINER_COUNT = "SELECT COUNT(id) AS ACTIVE_CONTAINERS_COUNT FROM " +
+    public static final String RUNNING_APPLICATION_VERSION_COUNT = "SELECT COUNT(id) AS ACTIVE_CONTAINERS_COUNT FROM " +
```
Comment (danushkaf) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/SQLQueryConstants.java]: containers are k8s specific. Why not make the variable ACTIVE_CONTAINERS_COUNT to RUNNING_APPLICATION_VERSION_COUNT and SQL query variable name to COUNT_RUNNING_APPLICATION_VERSIONS.

File: modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java [hunk 2]
```diff
@@ -1635,13 +1633,13 @@ public int getRunningApplicationVersionCount(Connection dbConnection, int tenant
...
         try {
-            preparedStatement = dbConnection.prepareStatement(SQLQueryConstants.GET_TENANT_RUNNING_CONTAINER_COUNT);
+            preparedStatement = dbConnection.prepareStatement(SQLQueryConstants.RUNNING_APPLICATION_VERSION_COUNT);
             preparedStatement.setInt(1, tenantId);
             preparedStatement.setString(2, cloudType);
             resultSet = preparedStatement.executeQuery();
             dbConnection.commit();
             if (resultSet.next()) {
-                appCount = resultSet.getInt("ACTIVE_CONTAINERS_COUNT");
+                appCount = resultSet.getInt(SQLQueryConstants.ACTIVE_CONTAINERS_COUNT);
```
Comment (danushkaf) [inline modules/components/org.wso2.appcloud.core/src/main/java/org/wso2/appcloud/core/dao/ApplicationDAO.java]: Better to name variables meaningful. Its about readability of the code and maintainability. Its running version count not app count.

File: modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag [hunk 3]
```diff
@@ -216,13 +216,13 @@ var log = new Log();
             } else {
                 //Since the user is trying to create a new application we check if the existing application count is
                 //less than the maximum amount of applications that can be created
-                var applicationCount = applicationModule.getRunningApplicationVersionCount();
-                var maxAppCount = applicationModule.getMaxInstanceCount();
-                if (applicationCount < maxAppCount) {
+                var RunningApplicationCount = applicationModule.getRunningApplicationVersionCount();
```
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag]: Its running application version count. Not running application count.

File: modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag [hunk 4]
```diff
@@ -216,13 +216,13 @@ var log = new Log();
...
+                var maxRunningAppCount = applicationModule.getMaxInstanceCount();
+                if (RunningApplicationCount < maxRunningAppCount) {
                     isAppCountLowerThanMax = true;
                 } else {
                     logMessage = "Application creation request not accepted because the application count has " +
```
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag]: message should get changed according to the change.

File: modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag [hunk 5]
```diff
@@ -262,7 +262,7 @@ var log = new Log();
                 response.content = logMessage;
                 return;
             } else {
-                logMessage = "Maximum number of applications that you can create is " + maxAppCount;
+                logMessage = "Maximum number of applications that you can create is " + maxRunningAppCount;
```
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag]: Its not the max no of applications anymore change it accordingly.

File: modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag [hunk 6]
```diff
@@ -357,7 +357,7 @@ var log = new Log();
             print(stringify(versions));
         } else {
             logMessage = "Application creation request not accepted because the application count has " +
```
Comment (danushkaf) [inline modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag]: Change the message here as well.

File: modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag [hunk 7]
```diff
@@ -342,9 +342,9 @@ var log = new Log();
             }
         }
     } else if(action == "startApplication"){
-        var applicationCount = applicationModule.getRunningApplicationVersionCount();
-        var maxAppCount = applicationModule.getMaxInstanceCount();
-        if (applicationCount < maxAppCount) {
+        var RunningApplicationCount = applicationModule.getRunningApplicationVersionCount();
```
Comment (SabraO) [inline modules/jaggeryapps/appmgt/src/site/blocks/application/application.jag]: variable name should start with a simple letter

**Pull request merged**